assets/scss/utilities/_hide.scss

```diff
@@ -28,9 +28,15 @@
 }
 
 .u-hidden\@medium {
-    display: block;
+    display: flex;
```

Comment:
If the classname is meant to hide elements, it should only add "display: none" when needed, without overwriting element's styles when it's visible